tests/integration/test_tcp_ipu_compatibility.py

```diff
@@ -19,6 +20,15 @@
 )
 
 
+@pytest.mark.abort_on_fail
```

Comment:
I would move this to conftest.py and make it an autouse fixture.

```
@pytest.fixture(autouse=True, scope='module')
async def setup_env(ops_test: OpsTest):
    # ensure update-status does not fire "too" quickly, else traefik will flip between
    # active/idle and maintenance: updating ingress configuration
    await ops_test.model.set_config(
        {"update-status-hook-interval": "60m", "logging-config": "<root>=WARNING; unit=DEBUG"}
    )
```

do this, and it will run before every module is run, so we don't need to repeat in each file